### PR TITLE
[Webpack] Add missing requires for popups

### PR DESCRIPTION
**Pull request by delphiactual**
This adds the missing requires for popups and renames the files accordingly
Also fixes the first problem in LoadoutBuilder

### CODE REVIEW

**Comment by delphiactual** [comment]
@ericnelson0 you might want to take a look at this right around [here](https://github.com/delphiactual/DIM/blob/07113f7358ec84ef3ee7d42f03a5ce2454305023/app/scripts/minmax/dimMinMax.controller.js#L569)

**Comment by bhollis** [comment]
Why do they have to be named `.template`?

**Comment by delphiactual** [comment]
has to do with `url-loader`

**Comment by joshhunt** [comment]
LGTM 👍 

https://github.com/DestinyItemManager/DIM/blob/webpack/config/webpack.js#L41

tl;dr: we need to apply a specific set of loaders to angular templates, so we need a unique filename for them.

We're using Webpack process two different 'types' of HTML files in two different ways

 1. `app/index.html`, which goes through just html-loader (which parses for `src="..."` and copies them over to the build dir). This has a relationship to `HtmlWebpackPlugin`.
 2. Angular templates for views and directives. We run this through html-loader, as well as file-loader (and extract-loader in the middle, but that's not terribly important here) so we can both parse and reference `src="..."` as well as copy the template itself to the build dir and get a reference to it's URL.

If the `app/index.html` file goes through the same loader set as `.templates.html` files, the index.html asset will actually just be the URL to that file (which doesnt make sense). 

To target both these types of files directly I made what I thought was the 'least wrong' choice of requiring angular templates to be in the form of `name.template.html`.

**Comment by delphiactual** [comment]
ok so the problem [here](https://github.com/delphiactual/DIM/blob/07113f7358ec84ef3ee7d42f03a5ce2454305023/app/scripts/minmax/dimMinMax.controller.js#L569) with LoadoutBuilder now is that tierString is still a  `SetIterator` and not a string now for some reason...

**Comment by joshhunt** [comment]
....huh. That sure is interesting.

**Comment by delphiactual** [comment]
alrighty LoadoutBuilder is working again
